src/community.c

```diff
@@ -3013,7 +3010,7 @@ static int igraph_i_community_multilevel_step(
         IGRAPH_ERROR("weights must be positive", IGRAPH_EINVAL);
     }
     if (gamma < 0.0) {
```

Comment:
I think this should still be changed to `resolution_parameter`, right?

Reply:
This function was only used internally by the main multilevel function (AFAIK) so I had left the *gamma*. But we could definitely change it to *resolution_parameter* too.